src/components/Panels/PropertiesPanel/PropertiesPanel.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div
+        id="moduleProperty"
+        class="moduleProperty noSelect effect1 properties-panel draggable-panel draggable-panel-css guide_2"
+    >
+        <PanelHeader :header-title="$t('simulator.panel_header.properties')" />
+        <div class="panel-body">
+            <div id="moduleProperty-inner">
+                <div id="moduleProperty-header">{{ panelBodyHeader }}</div>
+                <PanelType1 v-if="panelType == 1" />
+                <PanelType2 v-if="panelType == 2" />
+                <PanelType3
+                    v-if="panelType == 3"
+                    :key="toRaw(propertiesPanelObj)"
+                    :data="toRaw(propertiesPanelObj)"
+                />
+                <HelpButton
+                    :key="toRaw(propertiesPanelObj)"
+                    :data="toRaw(propertiesPanelObj)"
+                />
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { ref, toRaw } from '@vue/reactivity'
+import simulationArea from '#/simulator/src/simulationArea'
+import PanelHeader from '../Shared/PanelHeader.vue'
+import HelpButton from '../Shared/HelpButton.vue'
+import { onMounted } from 'vue'
+import {
+    checkPropertiesUpdate,
+    hideProperties,
+    prevPropertyObjGet,
+    prevPropertyObjSet,
+} from '#/simulator/src/ux'
+import { layoutModeGet } from '#/simulator/src/layoutMode'
+import PanelType1 from './PanelTypes/PanelType1.vue'
```

Comment:
Lets add proper names to `PanelType1` , `PanelType2`, `PanelType3`, & use alias for any import.

Reply:
Yes, I renamed it and also restructured the component folder structure.